### PR TITLE
Exposing the platform flag.  Looking for external config.

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**

### CODE REVIEW

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-BobbyEarl Any tips on how I can test this locally?

**Comment by Blackbaud-BobbyEarl** [comment]
Good question @Blackbaud-SteveBrush.  Quick update, working on the repo that'll be used to house additional configuration files for our supported platforms.  I'll leave another comment once that's ready to go.

**Comment by Blackbaud-BobbyEarl** [comment]
To best test this @Blackbaud-SteveBrush (aside from our test suite that's running), I suggest creating a SPA who's `package.json` has the following two devDependencies:

```
  "devDependencies": {
    "@blackbaud/skyux-builder": "blackbaud/skyux-builder#platform-flag",
    "@blackbaud/skyux-builder-config": "blackbaud/skyux-builder-config"
  },
```

From there, you can run `skyux test --platform vsts` and you should notice builder locating the vsts config file.  Let me know if there's more I can provide for you to go off of.To best test this @Blackbaud-SteveBrush (aside from our test suite that's running), I suggest creating a SPA who's `package.json` has the following two devDependencies:

```
  "devDependencies": {
    "@blackbaud/skyux-builder": "blackbaud/skyux-builder#platform-flag",
    "@blackbaud/skyux-builder-config": "blackbaud/skyux-builder-config"
  },
```

From there, you can run `skyux test --platform vsts` and you should notice builder locating the vsts config file.  Let me know if there's more I can provide for you to go off of.

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-BobbyEarl I get an error when I attempt to install `skyux-builder-config`, since it doesn't have an npm package.json. Thoughts?

**Comment by Blackbaud-BobbyEarl** [comment]
Updated the `skyux-builder-config` library @Blackbaud-SteveBrush

File: test/loader-assets.spec.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ describe('SKY UX assets Webpack loader', () => {
   let loader;
 
   beforeEach(() => {
-    mock('fs', {
+    mock('fs-extra', {
```
Comment (Blackbaud-SteveBrush) [inline test/loader-assets.spec.js]: Should probably run `mock.stopAll()` in the `afterEach` callback...

File: test/cli-utils-config-resolver.spec.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
+/*jshint jasmine: true, node: true */
+'use strict';
+
+const mock = require('mock-require');
+const logger = require('winston');
+
+describe('utils/config-resolver.js', () => {
+
+  function setup(command, globResult, internal, argv) {
+    mock('fs-extra', {
```
Comment (Blackbaud-SteveBrush) [inline test/cli-utils-config-resolver.spec.js]: This spec needs a `mock.stopAll()`...

**Pull request merged**